src/forge/rsync.py

```diff
@@ -102,8 +148,14 @@ def _rsync(config, ip):
                 continue
 
             for ip, _ in targets:
-                logger.info('Rsync destination is %s', ip)
-                rval = _rsync(config, ip)
+                if config.get('rsync_path'):
+                    logger.info('Rsync destination is %s', ip)
+                    rval += _rsync(config, ip)
+
+                if config.get('s3_path'):
+                    logger.info('S3 rsync destination is %s', ip)
+                    rval += _s3_rsync(config, ip)
+
```

Comment:
Can you add an else statement here in case the user does not have neither s3_path nor rsync_path?
I'm thinking we raise a warning with a message like: `Attempting to use rsync with no source. Maybe you forgot to add 'rsync_path' or 's3_path' in your config?`